server/server_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/allegro/bigcache"
+)
+
+var (
+	testBaseString = "http://bigcache.org"
+)
+
+func testCacheSetup() {
+	cache, _ = bigcache.NewBigCache(bigcache.Config{
+		Shards:             1024,
+		LifeWindow:         10 * time.Minute,
+		MaxEntriesInWindow: 1000 * 10 * 60,
+		MaxEntrySize:       500,
+		Verbose:            true,
+		HardMaxCacheSize:   8192,
+		OnRemove:           nil,
+	})
+}
+
+func TestGetWithNoKey(t *testing.T) {
```

Comment:
It will be great if we can run tests in parallel to ensure we do not rely on their order and they do not interfere with each other  https://golang.org/pkg/testing/#T.Parallel

Reply:
Fixed. Ran `go test -v .` in a loop 25 times to make sure.